### ISSUE TITLE
Feature: Allow specifying ingressClassName from values.yaml

### ISSUE THREAD

**Issue by tpoxa** [arttor/helmify#126]

**Comment by arttor** [comment]
Good idea! Probably the best way is to modify the ingress processor [pkg/processor/service/ingress.go](https://github.com/arttor/helmify/blob/main/pkg/processor/service/ingress.go#L16) in the following way:
1. Wrap template with `{{- if ...`
2. Add `ingress.enabled` to values.yaml
